liblangutil/SourceLocation.h

```diff
@@ -119,4 +119,24 @@ SourceLocation parseSourceLocation(
 /// Stream output for Location (used e.g. in boost exceptions).
 std::ostream& operator<<(std::ostream& _out, SourceLocation const& _location);
 
+
+/**
+ * Alternative, line-column-based representation for source locations.
+ * Both line and column are zero-based.
+ * If used as a range, the second location is considered exclusive.
+ * Negative values are invalid.
+ */
+struct LineColumn
+{
+	/// Line value, can be between zero and number of `\n` characters in the source file.
+	int line = {-1};
+	/// Column value, can be between zero and number of characters in the line (inclusive).
+	int column = {-1};
+
+	LineColumn() = default;
+	explicit LineColumn(std::tuple<int, int> const& _t): line{std::get<0>(_t)}, column{std::get<1>(_t)} {}
```

Comment:
I think this is actually not needed anymore.

Reply:
Yeah, I think it's not needed.